packages/ui/src/components/Alert/alert.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Alert as DefaultAlert, AlertProps as DefaultAlertProps } from 'antd'
+import classNames from 'classnames'
+
+type CondoAlertProps = {
+    type: 'success' | 'info' | 'warning' | 'error';
+}
+
+const ALERT_CLASS_PREFIX = 'condo-alert'
+
+export type AlertProps = Omit<DefaultAlertProps, 'type' | 'prefix' | 'prefixCls' | 'banner' | 'afterClose' | 'closable' | 'closeText' | 'closeIcon' | 'onClose'> & CondoAlertProps
+
+const Alert: React.FC<AlertProps> = (props) => {
+    const { type, className, ...rest } = props
+
+    const classes = classNames(
+        {
+            [`${ALERT_CLASS_PREFIX}-${type}`]: type,
+        },
+        className,
+    )
+
+    return (
+        <DefaultAlert
+            {...rest}
+            type = {type}
+            prefixCls={ALERT_CLASS_PREFIX}
+            className={classes}
```

Comment:
And what about icons? seems like now we're using custom icons here. Why not it's included here? 

P.S. Seems strange to see that almost all alerts are using info icon. You better to discuss it with Anton